src/test/java/Safron/NewBalanceUa/Tests/SignIn.java

```diff
@@ -0,0 +1,32 @@
+package Safron.NewBalanceUa.Tests;
+
+import BasePageAndTestInit.TestInit;
+import Safron.NewBalanceUa.Pages.NewBalanceHeaderViewElements;
+import Safron.NewBalanceUa.Pages.NewBalanceSignInViewElements;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class SignIn extends TestInit {
+
+    @DataProvider(name = "user info")
+    public Object[][] dpMethod() {
+        return new Object[][]{
+                {"elsafronitaz@gmail.com", "E8crmWqTtYU2GD7"}};
```

Comment:
Can you use @DataProvider a few more times?

Reply:
no